### ISSUE TITLE
Feature Request - specify 2.1 & short text encoder description

### ISSUE THREAD

**Issue by dalmackay** [TheLastBen/fast-stable-diffusion#943]
I see that v2 was replaced with 2.1 entirely but it's still called v2 in the colab. I think to reduce confusion it should be specified as "v2.1".

Also, The changes made to the text encoder description are even more ominous now, I know people would very much appreciate a short description of what it's doing, in layman's terms, so they can decide if and for how long they should use it for their specific purposes.

Thanks Ben, your work is massively appreciated.

**Comment by TheLastBen** [comment]
With the latest update, I separated the text_enc training from the UNet training, which helps give good results at low step count, but the text_enc steps should be kept low, no matter the dataset 350-500 steps is more than enough.
